lib/yang_wrappers.c

```diff
@@ -1137,6 +1137,25 @@ void yang_str2mac(const char *value, struct ethaddr *mac)
     (void)prefix_str2mac(value, mac);
 }
 
+struct yang_data *yang_data_new_date_and_time(const char *xpath, time_t uptime)
```

Comment:
I'd prefer to rename `uptime` to `time` only, since the `date-and-time` type can be used to represent other things other than uptimes.

Reply:
Changed `uptime` to `time` name.